systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,417 @@
+        final String namespaceName = StUtils.getNamespaceBasedOnRbac(NAMESPACE, extensionContext);
+        int topicsCount = 50;
+        int topicPartitions = 100;
+        String createAdminName = "create-admin-client";
+        String alterAdminName = "alter-admin-client";
+        String topicNamePrefix = classTopicPrefix + "-partitions";
+
+        setupKafkaInNamespace(extensionContext, clusterName, namespaceName);
+
+        KafkaBasicExampleAdminClient adminClientJob = new KafkaBasicExampleAdminClient.Builder()
+                .withAdminName(createAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(clusterName))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(topicsCount)
+                .withPartitions(topicPartitions)
+                .withNamespaceName(namespaceName)
+                .withTopicOperation("create")
+                .withAdditionalConfig(getAdminClientConfig(namespaceName))
+                .build();
+        resourceManager.createResource(extensionContext, true, adminClientJob.adminStrimzi().build());
+
+        Pod createPod = kubeClient().listPods("job-name", createAdminName).get(0);
+        PodUtils.waitUntilMessageIsInPodLogs(
+                createPod.getMetadata().getName(),
+                "org.apache.kafka.common.errors.ThrottlingQuotaExceededException: The throttling quota has been exceeded.",
+                Duration.ofMinutes(5).toMillis()
+        );
+        LOGGER.debug("=================\n" + kubeClient().logs(createPod.getMetadata().getName()));
+        JobUtils.deleteJobWithWait(namespaceName, createAdminName);
+
+        // Delete created topics (as they were created in random order, we have to use KafkaTopic instead of this client)
+        KafkaTopicList ktl = KafkaTopicResource.getAllKafkaTopicsWithPrefix(namespaceName, topicNamePrefix);
+        for (KafkaTopic kt : ktl.getItems()) {
+            cmdKubeClient().deleteByName(KafkaTopic.RESOURCE_SINGULAR, kt.getMetadata().getName());
+        }
+
+        // Throttling quota after performed 'alter' partitions on existing topic
+        int topicAlter = 20;
+        adminClientJob = new KafkaBasicExampleAdminClient.Builder()
+                .withAdminName(createAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(clusterName))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(topicAlter)
+                .withPartitions(1)
+                .withNamespaceName(namespaceName)
+                .withTopicOperation("create")
+                .withAdditionalConfig(getAdminClientConfig(namespaceName))
+                .build();
+        resourceManager.createResource(extensionContext, true, adminClientJob.adminStrimzi().build());
+
+        createPod = kubeClient().listPods("job-name", createAdminName).get(0);
+        PodUtils.waitUntilMessageIsInPodLogs(createPod.getMetadata().getName(), "All topics created");
```

Comment:
I guess that this message appear when the job will be successful. Wouldn't be better to wait for job to be successful then?